microsoft-edge/web-platform/user-agent-string.md

```diff
@@ -9,24 +9,112 @@ ms.prod: microsoft-edge
 keywords: microsoft edge, compatibility, web platform, user agent string, ua string, ua overrides
 ---
 
-# Microsoft Edge User Agent String (Desktop)  
+# Retrieving User Agent Data in Microsoft Edge 
 
-A user agent \(UA\) string is able to be used to detect what version of a specific browser is being used on a certain operating system.  Like other browsers, Microsoft Edge includes this information in the `User-Agent` HTTP header whenever it makes a request to a site.  It may also be accessed via JavaScript by querying the value of `navigator.userAgent`.  
+The User Agent \(UA\) can be used detect what version of a specific browser is being used on a certain operating system. Microsoft Edge offers two mechanisms for retrieving User Agent information: via User Agent Client Hints (recommended) and the legacy User Agent string. These can be accessed as follows: 

```

Comment:
The "User Agent" is the browser itself (e.g. Edge is a UA, Chrome is a UA, etc.). The "UA string" is one of the detection mechanisms, UA Client Hints is another.

Can we instead say something like, "Browsers provides mechanisms to detect brand and version information as well as to detect other device characteristics such as the host OS. Since the UA string has a long history of being a source of web site compatibility bugs, we recommend moving to a newer mechanism for such detection: UA Client Hints."

Reply:
Added.